src/rust/engine/src/externs.rs

```diff
@@ -187,17 +188,25 @@ pub fn unsafe_call(func: &Function, args: &[Value]) -> Value {
 lazy_static! {
   static ref EXTERNS: RwLock<Option<Externs>> = RwLock::new(None);
   static ref INTERNS: RwLock<Interns> = RwLock::new(Interns::new());
-  static ref LOGGER: FfiLogger = FfiLogger{};
 }
 
+// This is mut so that the max level can be set via set_externs, but so that a lock doesn't need to
+// be acquired every time we check whether we should log something.
+// Please don't mutate it.
+// Please.
+static mut LOGGER: FfiLogger = FfiLogger { level_filter: log::LevelFilter::Off };
```

Comment:
I found the wording here a little confusing. Maybe this would be better?
```
This is mut so that the max log level can be set via set_externs. But in order to avoid a performance hit, there is no lock guarding it. Please don't mutate it. Please.

Reply:
Reworded along those lines